src/gui/attributetable/qgsfieldconditionalformatwidget.h

```diff
@@ -0,0 +1,69 @@
+/***************************************************************************
+                          qgsfieldconditionalformatwidget.h
+                             -------------------
+    begin                :
+    copyright            :
+    email                :
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSFIELDCONDITIONALFORMATWIDGET_H
+#define QGSFIELDCONDITIONALFORMATWIDGET_H
+
+#include <QWidget>
+#include <QStandardItemModel>
+#include <QStandardItem>
+
+#include "ui_qgsfieldconditionalformatwidget.h"
+#include "qgsfieldcombobox.h"
+
+
+class GUI_EXPORT QgsFieldConditionalFormatWidget : public QWidget, private Ui::QgsFieldConditionalWidget

```

Comment:
Man that will teach me for coding at 12 in the morning. Thanks

On Fri, 14 Aug 2015 9:48 am Nyall Dawson <notifications@github.com> wrote:

> In src/gui/attributetable/qgsfieldconditionalformatwidget.h
> <https://github.com/qgis/QGIS/pull/2209#discussion_r37038505>:
>
> > + *   (at your option) any later version.                                   *
> > + *                                                                         *
> > + ***************************************************************************/
> > +
> > +#ifndef QGSFIELDCONDITIONALFORMATWIDGET_H
> > +#define QGSFIELDCONDITIONALFORMATWIDGET_H
> > +
> > +#include <QWidget>
> > +#include <QStandardItemModel>
> > +#include <QStandardItem>
> > +
> > +#include "ui_qgsfieldconditionalformatwidget.h"
> > +#include "qgsfieldcombobox.h"
> > +
> > +
> > +class GUI_EXPORT QgsFieldConditionalFormatWidget : public QWidget, private Ui::QgsFieldConditionalWidget
>
> Travis will fail until doxygen strings are added for this class
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qgis/QGIS/pull/2209/files#r37038505>.
>